### PR TITLE
chore(proxy): update librad to get new changes

### LINKED ISSUE #1041: Entity not found when navigating to maintainer profile
When tracking a peer project locally and clicking in the peer selector to navigate to the profile of the maintainer it show entity not found.

### PR DESCRIPTION
**Pull request by FintanH**
Some things were updated and fixed in librad including:
* Ref API changes
* Fetching of certifier identities
* untrack being idempotent

Fixes #1041

### CODE REVIEW

**Comment by FintanH** [comment]
Seems like some of my changes broke some shit:
```
thread 'can_fetch_project_changes' panicked at 'unable to fetch: Storage(Git(Error { code: -4, klass: 4, message: "failed to write reference \'refs/namespaces/hwd1yrebfcmxyug1u1zqi8kqneqgfkpbc4iwgxxx4eh5hnecxafy5ogp5py/refs/rad/ids/hwd1yrer4ws7jcct97117fwcyy6fq94xxpa9fgmkr8s73h68tp7uiakdg8y\': a reference with that name already exists." }))', coco/tests/replication.rs:176:9
```

This fails when we try to call `fetch_repo` in attempt to see if there's new changes.

File: proxy/coco/src/state.rs [hunk 1]
```diff
@@ -51,12 +57,31 @@ impl State {
...
+        Self {
+            api,
+            signer,
+            transport_results,
+        }
+    }
+
+    /// Ensure that we give the local transport some time to process any final tasks. See
+    /// [`transport::Results::wait`] for more information.
+    pub(crate) fn process_transport_result(&self) -> Result<(), Error> {
```
Comment (FintanH) [inline proxy/coco/src/state.rs:75]: @kim is this kosher? :eyes:

File: proxy/coco/tests/replication.rs [hunk 2]
```diff
@@ -330,6 +333,8 @@ async fn can_create_working_copy_of_peer() -> Result<(), Box<dyn std::error::Err
         rad.push(&[&format!("refs/heads/{}", project.default_branch())], None)?;
         commit_id
     };
+    assert!(results.wait(Duration::from_secs(3)).is_some());
```
Comment (FintanH) [inline proxy/coco/tests/replication.rs:336]: This always seems to timeout o.O

**Review by xla — APPROVED** [review]
😱 🛩 👋 🌰

**Review by rudolfs — APPROVED** [review]

**Pull request merged**